tests/phpunit/tests/theme/wpThemeJson.php

```diff
@@ -2502,6 +2502,83 @@ public function test_return_block_node_paths() {
 		$this->assertEquals( $expected, $block_nodes );
 	}
 
+	/**
+	 * This test covers `get_block_nodes` with the `$include_node_paths_only`
+	 * and `include_block_style_variations` options.
+	 */
```

Comment:
```suggestion
	 *
	 * @ticket 61858	 
	 */
```

Nit: Think we need a `@ticket` here.

Reply:
Good spotting. Updated to `62399`